executor/aggfuncs/func_count.go

```diff
@@ -0,0 +1,61 @@
+package aggfuncs
+
+import (
+	"github.com/juju/errors"
+
+	"github.com/pingcap/tidb/sessionctx"
+	"github.com/pingcap/tidb/util/chunk"
+)
+
+type baseCount struct {
+	baseAggFunc
+}
+
+type partialResult4Count struct {
```

Comment:
`type partialResult4Count = int64` is better

Reply:
I will fix it.